src/main/java/seedu/address/ui/ExpandedPersonCard.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+import java.util.List;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Log;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component that displays the full information of a {@code Person}.
+ */
+public class ExpandedPersonCard extends UiPart<Region> {
+
+    private static final String FXML = "ExpandedPersonListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
```

Comment:
Any reason this is public?

Reply:
Thanks for spotting this, I think private would be a better access modifier.